frontend/public/components/resource-quota.jsx

```diff
@@ -13,6 +13,18 @@ const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
 
+const compareResourceType = (left, right) => {
+  const regex = /\.|\//; //find object types delimited by '.' or '/'. Ex: '.cpu', '.memory', or '/persistentvolumeclaims'
+  left = left.match(regex) ? left.split(regex).reverse().join('') : left;
```

Comment:
You shouldn't need to check for matches. `split` will return an array with one value (the string) when there are no matches.

```suggestion
  left = left.split(regex).reverse().join('');
```